yarn.lock

```diff
@@ -1352,14 +1352,22 @@
     "@types/prop-types" "*"
     "@types/react" "*"
 
-"@types/react@*", "@types/react@^16.4.1":
+"@types/react@*":
```

Comment:
why do we have resolutions for `@types/react` twice? Could you delete these entries from yarn.lock and run yarn again to make sure that it finds only one. That's probably a reason of compilation errors, since it cannot assign type  from one `@types/react` to another.